downloadDirectory/src/test/java/downloadDirectory/TestProtegeExporter.java

```diff
@@ -0,0 +1,67 @@
+package downloadDirectory;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+
+import org.gk.persistence.MySQLAdaptor;
+import org.junit.Before;
+import org.junit.Test;
+import org.reactome.release.downloadDirectory.ProtegeExporter;
+
+@SuppressWarnings("static-method")
+public class TestProtegeExporter
+{
+	private static final String RELEASE_NUM = "70";
+	private static String host;
+	private static String name;
+	private static String user;
+	private static String password;
+	static final String PATH_TO_JAVA_ROOT = "/home/sshorser/workspaces/reactome/ReleaseNG/data-release-pipeline/downloadDirectory/";
```

Comment:
Can the `PATH_TO_JAVA_ROOT` be relative so the test can run when the repository is cloned on other machines?

Reply:
Hmm I can look into that...